dist/role/manifests/archives.pp

```diff
@@ -0,0 +1,6 @@
+#
+# Okra is a tiny VM (1vCPU/4GB RAM) on Oracle
```

Comment:
I thought that we already had a machine named `okra`.  Should this have a distinct name?
```suggestion
# Okra is a tiny VM (1vCPU/4GB RAM) on Oracle
```

Reply:
Good point